### ISSUE TITLE
Inspectdb: unique_together should be sorted

### ISSUE THREAD

**Issue by kevin-olbrich** [michiya/django-pyodbc-azure#100]
Hi,

we use inspectdb on a nightly run to create models from a productions database managed by a third party.
When we need a new table in our application, we copy the needed models from the inspectdb created file.

Models are generated every night and commited into our SCM. While table columns are in the same ordner on every run, unique_together isn't. Detected fields are random in order. This means that every run commits the same spec but with changed unique_together order:

```
 class Cpi6(models.Model):
@@ -17645,7 +17645,7 @@
     class Meta:
         managed = False
         db_table = 'CPV10'
-        unique_together = (('docentry', 'lineseq'), ('docentry', 'aftlinenum', 'ordernum'),)
+        unique_together = (('docentry', 'aftlinenum', 'ordernum'), ('docentry', 'lineseq'),)
 
 
 class Cpv11(models.Model):
@@ -20307,7 +20307,7 @@
     class Meta:
         managed = False
         db_table = 'CSV10'
-        unique_together = (('docentry', 'aftlinenum', 'ordernum'), ('docentry', 'lineseq'),)
+        unique_together = (('docentry', 'lineseq'), ('docentry', 'aftlinenum', 'ordernum'),)
 
 
 class Csv11(models.Model):
@@ -21089,7 +21089,7 @@
     class Meta:
         managed = False
         db_table = 'CSV5'
-        unique_together = (('absentry', 'wtcode', 'baseabsent', 'doc1lineno'), ('absentry', 'linenum'),)
+        unique_together = (('absentry', 'linenum'), ('absentry', 'wtcode', 'baseabsent', 'doc1lineno'),)
 
 
 class Csv6(models.Model):
@@ -22723,7 +22723,7 @@
     class Meta:
         managed = False
         db_table = 'DLN5'
-        unique_together = (('absentry', 'wtcode', 'baseabsent'), ('absentry', 'linenum'),)
+        unique_together = (('absentry', 'linenum'), ('absentry', 'wtcode', 'baseabsent'),)
```

Best would be to sort them by alphabetical order.

**Comment by kevin-olbrich** [comment]
Is this something that must be adjusted in django-pyodbc-azure or django (inspectdb)?

**Comment by kevin-olbrich** [comment]
This is still an issue on the latest release. This does not happen when using inspectdb for MySQL DBs.

**Comment by michiya** [comment]
I don't think it is a requirement for Django database adapters because they just assure to list unique_together columns in arbitrary order on inspection as you can see in the Django test code below.
https://github.com/django/django/blob/2.0.2/tests/inspectdb/tests.py#L230-L235

I tried to reproduce the issue at my end but I couldn't (inspectdb always lists unique_together columns in the same order on every run). I also took a look into the other standard Django backend codes but was unable to find any logic in them to list unique_together columns in a particular order on inspection. I don't know why you don't see the issue on MySQL but I guess it could potentially occur even when you work with the other standard Django database adapters.

**Comment by michiya** [comment]
I have added more strict ordering for introspection (dfb97d4). I hope this helps but I'm not sure if it really does or not because I'm still unable to reproduce the issue at my end.
